docs/contracts/overview/LSP17ContractExtension.md

```diff
@@ -14,6 +14,12 @@ The list of extensions for specific functions are stored under the specific data
 
 You can inherit the [`LSP17Extendable.sol`](../contracts/LSP17ContractExtension/LSP17Extendable.md) contract to create an extendable contract.
 
+To add an extension handler to the extendable contract you would have to add the handler's contract address as data value for the data key constructed as: 
+- 10 bytes, 20 hexadecimal characters - `LSP17Extension` mapping prefix
+- 20 bytes, 40 hexadecimal characters - the `bytes4` function selector rigth padded with 16 zero bytes.
+
+Now whenever a call is made the chosen function selector, the call will initially land in the fallback of the extendable contract. From there it will be forwarded to the retrieved handler contract address from the data key as constructed above.
```

Comment:
Suggest shortening a bit

```suggestion
Now whenever a call is made to this function selector on the extendable contract, it will land in its `fallback` function. From there, the calldata will be forwarded to the retrieved extension contract address set above.
```